### PR TITLE
Fix `geos:Geometry` to `ewkt` bug

### PR DESCRIPTION
**Pull request by lewiszlw**
Fixes https://github.com/georust/geozero/issues/202.

image

### CODE REVIEW

File: geozero/src/geos/geos_reader.rs [hunk 1]
```diff
@@ -101,12 +101,17 @@ fn process_coord_seq<P: GeomProcessor>(
...
     let n_coords = cs.size()?;
+    let cs_dims = cs.dimensions()?;
     for i in 0..n_coords {
         if multi {
             processor.coordinate(
                 cs.get_x(i)?,
                 cs.get_y(i)?,
-                Some(cs.get_z(i)?),
+                if matches!(cs_dims, geos::CoordDimensions::ThreeD) {
+                    Some(cs.get_z(i)?)
```
Comment (pka) [inline geozero/src/geos/geos_reader.rs:111]: Just out of curiosity - why doesn't a `cs_dims == geos::CoordDimensions::ThreeD` work?

**Review by pka — CHANGES_REQUESTED** [review]
Thanks fo the fix!

Could you please add your test case as well?

File: geozero/src/geos/geos_reader.rs (hunk 1 above)
Comment (lewiszlw) [reply geozero/src/geos/geos_reader.rs:111]: `==` works here. Changed.

**Review by pka — APPROVED** [review]
👍

**Pull request merged**